packages/terra-application-name/src/ApplicationHeaderName.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+
+import AppDelegate from 'terra-app-delegate';
+
+import styles from './ApplicationHeaderName.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The accessory element to be displayed next to the title.
+   * */
+  accessory: PropTypes.element,
+  /**
+   * The AppDelegate instance provided by the containing component. If present, its properties will propagate to the children components.
+   * */
+  app: AppDelegate.propType,
+  /**
+   * The title branding of the application name.
+   * */
+  title: PropTypes.string,
+};
+
+const ApplicationHeaderName = ({
+    accessory,
+    app,
+    title,
+    ...customProps
+  }) => {
+  const ApplicationHeaderNameClassNames = cx([
+    'application-header-name',
+    customProps.className,
+  ]);
+
+  return (
+    <div {...customProps} className={ApplicationHeaderNameClassNames}>
+      {!!accessory && <div className={cx('accessory')}>{accessory}</div>}
+      {!!title && <div className={cx('title')}><strong>{title}</strong></div>}
```

Comment:
If you are just using the `strong` element because it makes the text bold, it is likely not the element you want to use. This can be done via CSS in the title class. The `<strong>` element is for content that is of "strong importance," including things of great seriousness or urgency (such as warnings). Source: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong